spec/support/matchers/have_i18n_setting.rb

```diff
@@ -0,0 +1,17 @@
+require  'rspec/expectations'
+
+RSpec::Matchers.define :have_i18n_setting do |expected|
+  match do |actual|
+    method = expected.to_s + "_i18n"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.